src/main/java/school/hei/haapi/endpoint/rest/security/SecurityConf.java

```diff
@@ -55,6 +55,76 @@ public AuthenticationManager authenticationManager() {
     return new ProviderManager(authProvider);
   }
 
+  RequestMatcher MANAGER_AUTHORIZATION_PATH =
```

Comment:
rename to MANAGER_ONLY_REQUEST_MATCHER

as static variable if possible and private!!!


do the same for the others.

only selfMatchers can't be static as they need authResourceProvider field